mail/api.py

```diff
@@ -280,3 +292,80 @@ def send_course_team_email(  # pylint: disable=too-many-arguments
             raise_for_status=raise_for_status,
         )
         return response
+
+
+def send_automatic_emails(program_enrollment):
+    """
+    Send all automatic emails which match the search criteria for a program enrollment
+
+    Args:
+        program_enrollment (ProgramEnrollment): A ProgramEnrollment
+    """
+    percolate_queries = search_percolate_queries(program_enrollment.id)
+    automatic_emails = AutomaticEmail.objects.filter(
+        query__in=percolate_queries,
+        enabled=True,
+    ).exclude(sentautomaticemail__user__programenrollment=program_enrollment)
```

Comment:
I don't think there's a use case for a user receiving two similar emails from two different programs, though if there was we should change `SentAutomaticEmail` to connect with `ProgramEnrollment` instead. For right now there will not be an `AutomaticEmail` which matches more than one program because the learner search filters on a single program. What do you think @pdpinch?